spec.html

```diff
@@ -1585,158 +1585,121 @@ <h1>Number::bitwiseNOT ( _x_ )</h1>
+            1. Return 𝔽(_r_).
+          </emu-alg>
           <emu-note>
             <p>The result of _base_ `**` _exponent_ when _base_ is *1*<sub>𝔽</sub> or *-1*<sub>𝔽</sub> and _exponent_ is *+&infin;*<sub>𝔽</sub> or *-&infin;*<sub>𝔽</sub>, or when _base_ is *1*<sub>𝔽</sub> and _exponent_ is *NaN*, differs from IEEE 754-2019. The first edition of ECMAScript specified a result of *NaN* for this operation, whereas later versions of IEEE 754-2019 specified *1*<sub>𝔽</sub>. The historical ECMAScript behaviour is preserved for compatibility reasons.</p>
           </emu-note>
         </emu-clause>
 
         <emu-clause id="sec-numeric-types-number-multiply" oldids="sec-applying-the-mul-operator">
           <h1>Number::multiply ( _x_, _y_ )</h1>
-          <p>The abstract operation Number::multiply takes arguments _x_ (a Number) and _y_ (a Number). It performs multiplication, producing the product of _x_ and _y_, as determined by the rules of IEEE 754-2019 binary double-precision arithmetic:</p>
-          <ul>
-            <li>
-              If either operand is *NaN*, the result is *NaN*.
-            </li>
-            <li>
-              The sign of the result is positive if both operands have the same sign, negative if the operands have different signs.
-            </li>
-            <li>
-              Multiplication of an infinity by a zero results in *NaN*.
-            </li>
-            <li>
-              Multiplication of an infinity by an infinity results in an infinity. The sign is determined by the rule already stated above.
-            </li>
-            <li>
-              Multiplication of an infinity by a finite non-zero value results in a signed infinity. The sign is determined by the rule already stated above.
-            </li>
-            <li>
-              In the remaining cases, where neither an infinity nor *NaN* is involved, the product is computed and rounded to the nearest representable value using IEEE 754-2019 roundTiesToEven mode. If the magnitude is too large to represent, the result is then an infinity of appropriate sign. If the magnitude is too small to represent, the result is then a zero of appropriate sign. The ECMAScript language requires support of gradual underflow as defined by IEEE 754-2019.
-            </li>
-          </ul>
+          <p>The abstract operation Number::multiply takes arguments _x_ (a Number) and _y_ (a Number). It performs multiplication, producing the product of _x_ and _y_. It performs the following steps when called:</p>
+          <emu-alg>
+            1. If _x_ is *NaN* or _y_ is *NaN*, return *NaN*.
+            1. If _x_ is *+&infin;*<sub>𝔽</sub> or _x_ is *-&infin;*<sub>𝔽</sub>, then
+              1. If _y_ is *+0*<sub>𝔽</sub> or _y_ is *-0*<sub>𝔽</sub>, return *NaN*.
+              1. If _y_ is positive, return _x_.
+              1. If _x_ is *+&infin;*<sub>𝔽</sub>, return *-&infin;*<sub>𝔽</sub>. Otherwise, return *+&infin;*<sub>𝔽</sub>.
```

Comment:
I'd prefer the `Otherwise` get its own line, here. Putting it inline obscures that there are four independent cases: y is 0, y is positive, x is +∞, otherwise (i.e. x is -∞).

Reply:
Also it's probably clearer to write `Return -_x_.`, rather than this branch.